app/containers/HomeContainer/index.js

```diff
@@ -144,6 +169,27 @@ export function HomeContainer({
           onSearch={searchText => debouncedHandleOnChange(searchText)}
         />
       </CustomCard>
+
+      <CustomCard
+        title={intl.formatMessage({ id: 'repo_search' })}
+        maxwidth={500}
+      >
+        <Text marginBottom={10} id="get_repo_details" />
+        <Search
```

Comment:
i see the same code is used somewhere else. please get rid of this duplication

Reply:
i remove this.